test/test.ts

```diff
@@ -28,8 +33,48 @@ describe("Test Excel Custom Functions", function () {
         const devServerCmd = `npm run dev-server -- --config ./test/webpack.config.js`;
         const devServerPort = parseNumber(process.env.npm_package_config_dev_server_port || 3000);
             await startDebugging(manifestPath, AppType.Desktop, toOfficeApp(host), undefined, undefined, 
-                devServerCmd, devServerPort, undefined, undefined, undefined, false /* enableDebugging */);
-    }),
+                devServerCmd, devServerPort, undefined, undefined, undefined, true /* enableDebugging */);
+    });
+    describe("Test Debugger", function () {
```

Comment:
I will try to work on the structure given the suggestion that you gave, but I would to discuss the implications of putting the debugger test completely detached from the normal execution tests.

The executions tests open excel, which is needed for the debugging test. I would like to only open excel one time that's why a structure like the following would be reasonable:

```
Tests Wrap-up (Should start and close excel)
  Custom Function execution
     Simple custom functions
     Streaming custom functions
  Custom Function debugging
     Basic debugging validation  (this is the test you're essentially writing)       
     Breakpoints
     Variable evaluation
     Call stack
     Exceptions```

Reply:
While you could try to do this with just one invocation of Excel, I think you'll likely need/want to open Excel more than once. The debugging start and stop can be structured so it can be done more than once and this will help not only this specific testing, but the reliability of the tests altogether. The problem when you have a test that does everything in one shot is that it can be difficult to diagnose when something goes wong -- because it does so much -- and potentially flaky. Once you've built out the support to do the basic stuff, it can then be much easier to leverage to add coverage.